attest/enclave-api/src/lib.rs

```diff
@@ -103,7 +103,7 @@ pub struct PlaintextClientRequest {
 /// The response to a request for a new report. The enclave will expect the
 /// QuoteNonce to be used when the report is quoted, and both the quote and
 /// report to be returned to the enclave during the verify_quote() phase.
-#[derive(Clone, Debug, Deserialize, Hash, Eq, Ord, PartialEq, PartialOrd, Serialize)]
+#[derive(Clone, Debug, Deserialize, Hash, Eq, PartialEq, Serialize)]
```

Comment:
@eranrund This is the higher level struct that wanted Ord and PartialOrd.

I'm not opposed to adding them back to the types, I mainly removed them as they didn't appear to be needed and they weren't implemented in `mc-sgx-core-types`